Gopkg.toml

```diff
@@ -21,6 +21,10 @@ required = [
   name = "github.com/stretchr/testify"
   version = "1.2.2"
 
+[[constraint]]
+  name = "github.com/kubernetes-sigs/kube-batch"
+  version = "v0.3"
```

Comment:
let's use 0.4

Reply:
When using using v0.4 of kube-batch, there is dependency conflict for k8s.io/api, which in kubeflow, v1.11.2 is used but in kube-batch 1.13.2 is used, that is why v0.3 is used for kube-batch

Reply:
@richardsliu , any plan to upgrade k8s version? 1.14 is going to be release, and upstream only support three versions (1.12, 1.13, 1.14) until LTS.

Reply:
With v1beta2, the subresource status requires 1.13.